src/repository/exercisesRepository.ts

```diff
@@ -325,6 +325,426 @@ export const db: Schema = {
+      Tu: 'conheces',
+      'Ela/Ele/Você': 'conhece',
+      Nós: 'conhecemos',
+      'Eles/Elas/Vocēs': 'conhecem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to solve',
+      infinitive: 'resolver',
+      Eu: 'resolvo',
+      Tu: 'resolves',
+      'Ela/Ele/Você': 'resolve',
+      Nós: 'resolvemos',
+      'Eles/Elas/Vocēs': 'resolvem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to go down',
+      infinitive: 'descer',
+      Eu: 'desço',
+      Tu: 'desces',
+      'Ela/Ele/Você': 'desce',
+      Nós: 'descemos',
+      'Eles/Elas/Vocēs': 'descem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to warm up',
+      infinitive: 'aquecer',
+      Eu: 'aqueço',
+      Tu: 'aqueces',
+      'Ela/Ele/Você': 'aquece',
+      Nós: 'aquecemos',
+      'Eles/Elas/Vocēs': 'aquecem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to live',
+      infinitive: 'viver',
+      Eu: 'vivo',
+      Tu: 'vives',
+      'Ela/Ele/Você': 'vive',
+      Nós: 'vivemos',
+      'Eles/Elas/Vocēs': 'vivem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to run',
+      infinitive: 'correr',
+      Eu: 'corro',
+      Tu: 'corres',
+      'Ela/Ele/Você': 'corre',
+      Nós: 'corremos',
+      'Eles/Elas/Vocēs': 'correm',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to learn',
+      infinitive: 'aprender',
+      Eu: 'aprendo',
+      Tu: 'aprendes',
+      'Ela/Ele/Você': 'aprende',
+      Nós: 'aprendemos',
+      'Eles/Elas/Vocēs': 'aprendem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to forget',
+      infinitive: 'esquecer',
+      Eu: 'esqueço',
+      Tu: 'esqueces',
+      'Ela/Ele/Você': 'esquece',
+      Nós: 'esquecemos',
+      'Eles/Elas/Vocēs': 'esquecem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to use',
+      infinitive: 'usar',
+      Eu: 'uso',
+      Tu: 'usas',
+      'Ela/Ele/Você': 'usa',
+      Nós: 'usamos',
+      'Eles/Elas/Vocēs': 'usam',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to have lunch',
+      infinitive: 'almoçar',
+      Eu: 'almoço',
+      Tu: 'almoças',
+      'Ela/Ele/Você': 'almoça',
+      Nós: 'almoçamos',
+      'Eles/Elas/Vocēs': 'almoçam',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to phone',
+      infinitive: 'telefonar',
+      Eu: 'telefono',
+      Tu: 'telefonas',
+      'Ela/Ele/Você': 'telefona',
+      Nós: 'telefonamos',
+      'Eles/Elas/Vocēs': 'telefonam',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to phone',
+      infinitive: 'telefonar',
+      Eu: 'telefono',
+      Tu: 'telefonas',
+      'Ela/Ele/Você': 'telefona',
+      Nós: 'telefonamos',
+      'Eles/Elas/Vocēs': 'telefonam',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to pay',
+      infinitive: 'pagar',
+      Eu: 'pago',
+      Tu: 'pagas',
+      'Ela/Ele/Você': 'paga',
+      Nós: 'pagamos',
+      'Eles/Elas/Vocēs': 'pagam',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to take',
```

Comment:
yes, but related to food (I think), because if you want to say "I will take this sofa home", you cannot user "tomar" but would be "levar" = "bring"

Reply:
I added levar